functions/index.js

```diff
@@ -31,14 +31,12 @@ admin.initializeApp({
 
 const access = require("./access");
 const auth = require("./auth");
-const payment = require("./payment");
 const stats = require("./stats");
 const venue = require("./venue");
 const video = require("./video");
 
 exports.access = access;
 exports.auth = auth;
-exports.payment = payment;
```

Comment:
Have we ensured that we have removed all of the frontend app functions that were calling `payment-*` methods?

Reply:
fixed in ebe967cd50437f138f6685ec3ce64daef7abeb40